test/router/path_trie_test.dart

```diff
@@ -502,5 +502,119 @@ void main() {
         expect(result.parameters, equals({#id: '456'}));
       });
     });
+
+    group('Use mapping', () {
+      test(
+          'Given a value at root, '
+          'when use is applied to root, '
+          'then the value is transformed', () {
+        final trie = PathTrie<int>();
+        final root = NormalizedPath('/');
+        trie.add(root, 1);
+        trie.use(root, (final i) => i * 2);
+        expect(trie.lookup(root)?.value, 2, reason: 'doubled');
```

Comment:
Suggestion: Not entirely sure how the reason is built but I believe this would give a better error message right?

```suggestion
        expect(trie.lookup(root)?.value, 2, reason: 'Value should have been doubled');
```

Reply:
Apply for the other tests with only `doubled` as well.

Reply:
You are probably correct. I should look it up though and try to memorize for the future.

Reply:
What about the slightly shorter `Should double`. Looks like this on failure:
```
00:00 +37 -1: PathTrie<T> Use mapping Given two tries where one is attached to the other, when use is applied to the root of the child, then only attached trie values are transformed [E]
  Expected: <200>
    Actual: <100>
  Should double
  
  package:matcher                        expect
  test/router/path_trie_test.dart 641:9  main.<fn>.<fn>.<fn>
  
```